scout/adapter/mongo/query.py

```diff
@@ -850,6 +852,25 @@ def secondary_query(self, query, mongo_query, secondary_filter=None):
                     fusion_caller_query.append({caller: "Pass"})
                 mongo_secondary_query.append({"$or": fusion_caller_query})
 
+            if criterion == "clinsig_onc":
```

Comment:
Looking forward to `match`! 😁

Reply:
Perhaps when we refactor the code for the query - we really should - we won't even need all these ifs any more